### PR TITLE
LPS-196022 Adds a page with collection fragments

### PR DESCRIPTION
**Pull request by jkappler**
## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-196022)

The goal of the task is to add a page with collections to DataFactory so that performance tests can be executed on it

## Change summary:

* Modify DataFactory class to add a page with collections. We create models for AssetListEntry and related entities to achieve this

## Test Scenario

* Fetch the pull request
* Go to benchmark folder in your source code for liferay portal
* To make it easier and faster to test, you can open benchmark.properties and change sample.sql.max.commerce.group.count to 0 and sample.sql.max.group.count to 1
* Execute ant build-sample-sql
* After this finishes, import sample-mysql.sql into your database
* Start the portal
* Go to Search and reindex BlogsEntry (In Apps Menu > Control Panel > Search)
* Go to Apps Menu > Control Panel > Sites, and go to the Pages of Site 1
* In the navigation menu, click on collection_page
* You will see 21 collections displaying the blogs of the site

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java [hunk 1]
```diff
@@ -4579,6 +4641,39 @@ public LayoutPageTemplateStructureModel newLayoutPageTemplateStructureModel(
...
+				String templateFileName, String templateFolder)
+		throws Exception {
+
+		LayoutPageTemplateStructureRelModel
+			layoutPageTemplateStructureRelModel =
+				newLayoutPageTemplateStructureRelModel(
+					layoutModel, layoutPageTemplateStructureModel,
+					fragmentEntryLinkModels, templateFileName, templateFolder);
+
+		String data = StringUtil.replace(
```
Comment (ealonso) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:4660]: Hey @jkappler, here we can use:

```
StringUtil.replace(
			layoutPageTemplateStructureRelModel.getData(),
			new String[] {
				"${assetListEntryClassNameId}", "${assetListEntryClassPK}",
				"${assetListEntryTitle}"
			},
			new String[] {
				String.valueOf(getClassNameId(AssetListEntry.class)),
				String.valueOf(assetListEntryModel.getAssetListEntryId()),
				String.valueOf(assetListEntryModel.getTitle())
			})
```

What do you think?

Comment (jkappler) [reply modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:4660]: sounds great, will change it

**Comment by ealonso** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**